### DISCUSSION TITLE
Unlink x and y values on scale property

### DISCUSSION THREAD

**Discussion by Finnein** [keiwando/toonsquid#351] [Ideas]
Hi!
It would do it much easier to stretch and squeeze objects (text, shapes, images etc) if there were an option to unlink x and y values from each other on scale property when dragging on the handles on object on the canvas. As it is now you, can only scale the object up and down (x and y values are linked together).

**Comment by DinoAwakening** [comment]
https://github.com/keiwando/toonsquid/assets/120270913/40d017f8-59d6-4cba-a519-865bb98a3ed9

**Reply by Finnein** [reply]
I thought that the only way to change x and y scale values unlinked to each other was by changing the values in the properties panel. But that was because the transform mode is by default on Uniform. By changing the mode to Freeform  you can change the scale values unlinked to each other by dragging the control points on the canvas. Thanks!
